### ISSUE TITLE
[Feature]: Assembler Recipes

### ISSUE THREAD

**Issue by MaGeBZz** [ThePansmith/Monifactory#1040]
### Suggestion

I think the pack would be a bit nicer if all things get assembler recipes to avoid crafting especially with tools

Things that I think about would be:
- Wooden Widget Left/Right Part
- Stone Widget Up/Down Side
- Redstone Conduit
- Pressurized Fluid Conduit
- ME Interface (AE2)
- Blank Pattern (AE2)
- Basic Card (AE2)
- Every other AE2 card as well
- 1k ME Storage Component
Maybe even getting recipes with LuV and ZPM circuits to make 32/64 from the 1k storage components
Maybe even more advanced recipes for UV->MAX circuits to use them for storage components as well if you wanna use it
Getting more ways with higher circuits to make components better is a good way imo to be able to make higher cells if you want to before getting kinda creative in endgame
- Formation/Annihilation Core (AE2)
- Quartz Glass (AE2)
- ME Drive (AE2)
- ME Chest (AE2)
- Molecular Assembler (AE2)
- Fluix ME Smart Cable (AE2)
- Illuminated Panel (AE2)
- ME Level Emitter (AE2)
- ME Pattern Provider (AE2)
- ME Wireless Connector (AE2)
- Wireless Receiver (AE2)

I'm pretty early in the pack rn but I would update this thing if I see something else or people could comment on things they would like to see assembler recipes too!

**Comment by Yokoharu** [comment]
You know that if something doesn't have an assembler recipe but a crafting recipe you can automate it in Molecular Assembler, right?

**Comment by MaGeBZz** [comment]
@Yokoharu I know that but you know that greg machines are faster than almost anything else in modded minecraft, right? And it's a greg pack, so having assembler recipes is pretty essential imo and it's just not consistently if some things got recipes and others don't although things are pretty similar sometimes

**Comment by Yokoharu** [comment]
If you build proper Molecular Assembler cluster it won't be slower than machines, it will be faster actually

**Comment by MaGeBZz** [comment]
I don't think so with a machine with max parallel + 2x biggest energy hatch but even if it is like that there should be a greg way anyways imo

**Comment by Kasra-G** [comment]
> - Wooden Widget Left/Right Part
> - Stone Widget Up/Down Side

These are balanced to be manual craft only, since they are used to get Monicoins.

> if all things get assembler recipes

It's a huge pain to add these recipes. Chances are the bottleneck is not the molecular assembler for most of these anyway. The current behavior of Interfaces crafting in parallel with adjacent molecular assemblers, along with acceleration cards, is all that is needed.

Nomifactory does have a molecular assembler multiblock, but I'm not too sure how often that is used. Most items will end up being passive crafted post-tank anyway, which is probably when you would have the biggest crafts.

**Comment by Xefyr0** [comment]
With the Assembler Matrix from ExtendedAE (#1125) this is no longer an issue.
